app/consistent_apphash_test.go

```diff
@@ -197,6 +466,44 @@ func queryAccountInfo(capp *app.App, accs []string, kr keyring.Keyring) []blobfa
 	return infos
 }
 
+func getAddress(account string, kr keyring.Keyring) sdk.AccAddress {
+	rec, err := kr.Key(account)
+	if err != nil {
+		panic(err)
+	}
+	addr, err := rec.GetAddress()
+	if err != nil {
+		panic(err)
+	}
+	return addr
+}
```

Comment:
I thought we already had a function like this in test util package

Reply:
on main we do